### DISCUSSION TITLE
Problem With Webradio Cover Art

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1476] [Bug Reports]
@rern
The BBC have changed their stream URLs again, and rAudio does not display the logo image with the replacement address format. The image is saved to the folder under the new URL, but is ignored. I think this might be because of the % character, but do not know how to prove this.

Example URL: http://as-hls-ww-live.akamaized.net/pool_904/live/ww/bbc_radio_three/bbc_radio_three.isml/bbc_radio_three-audio%3d96000.norewind.m3u8

Would it be possible to have the image title as the station name, rather than the URL?

**Comment by R101G** [comment]
OK, changing %3d to = solves the problem. The image title gets changed automatically but the station stream doesn't, which looks like the cause of the problem.

**Comment by klassikmann** [comment]
@R101G 
You find additional informations regarding the change of the BBC Radio URLs at https://garfnet.org.uk/cms/tables/radio-frequencies/internet-radio-player/

**Reply by R101G** [reply]
@klassikmann 
The rewind list looks OK, but it is worth doing a search and replace on the no-rewind entries if you want to use any of those URLs in the rAudio library with a logo as cover art. Change _%3d to =_ and, if you are in the UK, change both _ww_ instances to _uk_. Apparently some rights-restricted content might get blocked otherwise. You can also change the 96000 to 320000.

Looks like this is the reason the BBC wants to stop us using anything except their Sounds player:
https://blog.james.cridland.net/what-does-the-bbc-know-about-me-d52d26dc9f77
